src/bosh-director/lib/bosh/director/jobs/update_release.rb

```diff
@@ -242,29 +243,29 @@ def process_jobs(release_dir)
         existing_jobs = []
         manifest_jobs = @manifest['jobs'] || []
 
-        manifest_jobs.each do |job_meta|
+        manifest_jobs.each do |manifest_job|
           # Checking whether we might have the same bits somewhere
-          @release_version_model.templates.select { |t| t.name == job_meta['name'] }.each do |tmpl|
-            next unless tmpl.fingerprint != job_meta['fingerprint']
+          @release_version_model.templates.select { |t| t.name == manifest_job['name'] }.each do |release_job|
+            next unless release_job.fingerprint != manifest_job['fingerprint']
 
             raise(
               ReleaseExistingJobFingerprintMismatch,
-              "job '#{job_meta['name']}' had different fingerprint in previously uploaded release '#{@name}/#{@version}'",
+              "job '#{manifest_job['name']}' had different fingerprint in previously uploaded release '#{@name}/#{@version}'",
             )
           end
 
-          jobs = Models::Template.where(fingerprint: job_meta['fingerprint']).all
+          job_models = Models::Template.where(fingerprint: manifest_job['fingerprint']).all
```

Comment:
About `Models::Template` representing a “job”, I’ve updated the comment of the class there:
https://github.com/cloudfoundry/bosh/pull/2479/files#diff-56d25d21526860acc6209368812592a1819f9db61fd20db419dabdca49173433R2-R5

I’ve thought this to be sufficient for the moment. Tell me what you think about this.

For a first refactoring, I’ve also deliberately chosen not to rename `Models::Template` in order to avoid renaming tables, and broader and more risky impacts. But it would be good to do it at some point.